stdlib/2and3/traceback.pyi

```diff
@@ -113,6 +113,9 @@ if sys.version_info >= (3, 5):
                     frame_gen: Generator[Tuple[FrameType, int], None, None],
                     *, limit: Optional[int] = ..., lookup_lines: bool = ...,
                     capture_locals: bool = ...) -> StackSummary: ...
+        # TODO(https://bugs.python.org/issue34648): relax "a_list" to
+        # Iterable[Union[_PT, FrameSummary]]... and then at some later point
+        # drop support for _PT.
         @classmethod
-        def from_list(cls, a_list: List[_PT]) -> StackSummary: ...
+        def from_list(cls, a_list: List[Union[_PT, FrameSummary]]) -> StackSummary: ...
```

Comment:
This will probably not work as expected due to the invariance of `List`. (Although when I just tested it, mypy for some reason interpreted `List[Union[_PT, FrameSummary]]` as `List[Any]`.) This should be `Union[List[_PT], List[FrameSummary]]`.

Reply:
Why not just `Iterable` or `Sequence`? Even if the documentation says "list", we routinely accept a wider type in stubs; documentation is often written in a more informal style.

Reply:
@JelleZijlstra: [I agree!](https://bugs.python.org/issue34648)

... but however routine it is, it didn't strike me as a good idea for `typeshed` to describe as acceptable a wider type than what a standard library module's maintainers officially describe as supported.

Reply:
I think both are acceptable. Personally, I would go with a `List` as that is what the stdlib maintainers seem to prefer, but accepting an `Iterable` is fine, too. The latter has the advantage of being covariant.